### DISCUSSION TITLE
How to change undrline color for some situations

### DISCUSSION THREAD

**Discussion by amoktar** [microsoft/vscode-pylint#509] [Q&A] [answered]
Hello, 
Thanks for this awesome repo.

How can I change the color of the underline for situations like `Missing function or method docstringPylint`
which gives me a blue underline that is confilicting with other setting I have with vscode 

Thanks

**Comment by karthiknadig** [comment, marked as answer]
You can use `pylint.severity` setting to change that. replace `error-name` with either name or code.
```
{ 
    "convention": "Information",
    "error": "Error",
    "fatal": "Error",
    "refactor": "Hint",
    "warning": "Warning",
    "info": "Information",
    "<error-name>": "Warning"
}
```

Error = Red, Warning = Yellow, Information = Blue, Hint = Grey Dots.

**Reply by amoktar** [reply]
Thanks @karthiknadig 
I appreciate it
